Directory.Build.props

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <LangVersion>7.2</LangVersion>
     <DefaultItemExcludes>$(DefaultItemExcludes);*.DotSettings;*.ncrunchproject</DefaultItemExcludes>
+    <DebugType>portable</DebugType>
```

Comment:
For my understanding, why did you choose portable and not embedded?

Reply:
That's more of a personal preference. That way you can actually choose if you want to ship the debug info or not along with the binaries. This could matter for people who try to minimize their product's download size.
Both options should work the same, functionally speaking.

Reply:
Ok, let's run with this and we can always change to embedded if required